docs/cli.md

```diff
@@ -30,50 +30,118 @@ $ sematic run examples/mnist/pytorch
 Since Sematic makes sure not to include unnecessary heavy dependencies, you may
 be prompted to install those needed by the example you want to run.
 
-
 ### User settings
 
-Certain Sematic functionalities need certain settings variables to be set. For
-example, `SnowflakeTable` needs your Snowflake credentials to be able to query
-your data for you.
+Certain Sematic functionalities need certain settings variables to be set.
+These settings control which backend Server will be used when you issue
+commands, and other user-specific configurations. For example, `SnowflakeTable`
+needs your Snowflake credentials to be able to query your data for you.
 
 You can set user settings as follows:
 
 ```shell
 $ sematic settings set VAR VALUE
 ```
 
-For example
+For example:
 
 ```shell
 $ sematic settings set SNOWFLAKE_USER "foobar"
 ```
 
-Then you can check your stored settings with
+Then you can check your stored settings with:
 
 ```shell
 $ sematic settings show
-Active settings:
+Active user settings:
 
 SNOWFLAKE_USER: foobar
 ```
 
-These settings are simply store in the `/.sematic/settings.yaml` file on
-your machine.
+These settings are simply stored in the `~/.sematic/settings.yaml` file on
+your machine (the "default" qualifier is a placeholder for a future feature):
+
+```shell
+$ cat ~/.sematic/settings.yaml
+default:
+  SNOWFLAKE_USER: foobar
+```
 
 You can always override them at runtime with an environment variable:
 
 ```shell
 $ SNOWFLAKE_USER="notfoobar" sematic settings show
-Active settings:
+Active user settings:
 
 SNOWFLAKE_USER: notfoobar
 ```
 
 If you'd like, you can even change which directory Sematic uses to hold its
 settings by setting the environment variable `SEMATIC_CONFIG_DIR`. This can
-be either a relative path (it will be treated as relative to your home directory)
-or an absolute path.
+be either a relative path (it will be treated as relative to your home
+directory) or an absolute path.
+
+This is the full list of supported user settings:
+- `SEMATIC_API_ADDRESS`: the address of the Sematic server to use for pipeline
+  executions
+- `SEMATIC_API_KEY`: the user's personal API key, used to associate pipeline
+  submissions
+- `SNOWFLAKE_USER`: the Snowflake user
```

Comment:
nit: I would add a note to these saying that the snowflake settings are only required if you're using snowflake in your pipeline

Reply:
Done.